spigot/src/main/java/org/geysermc/floodgate/util/ClassNames.java

```diff
@@ -93,7 +103,7 @@ public class ClassNames {
 
         String nmsPackage = SPIGOT_MAPPING_PREFIX + '.';
 
-
+        
```

Comment:
Spaces not needed

Reply:
https://github.com/GeyserMC/Floodgate/pull/449/commits/2c52392451dc6af607b9b2194f6f460ecaff96a8#diff-c9d452f82010aef19e8cd407cb82aa21dff9a8fc01101c649ae2acb6e833b7e5R106